charts/polkadot-introspector/templates/deployment.yaml

```diff
@@ -52,6 +52,7 @@ spec:
               {{- end }}
               {{- end }}
               --ws {{ join "," .Values.introspector.rpcNodes }} \
+              -v \
```

Comment:
Let's add a condition here to allow us to disable and enable verbose logging via values.yaml. We can have it as:

```
introspector.verboseLogging: false
```

Ideally, it should be false by default then we will just override it in our own value file.